app/src/main/java/com/globant/practice/presentation/view/activity/HomeActivity.java

```diff
@@ -47,4 +76,51 @@ protected void onPause() {
         super.onPause();
         presenter.detachView();
     }
+
+    /**
+     * Renders the view items depending of the view state
+     *
+     * @param homeViewState State of the view
+     */
+    @Override
+    public void render(HomeViewState homeViewState) {
+        if (homeViewState.isLoading()) {
+            fetchUserIndicator.show();
+        } else if (homeViewState.getUsers() != null) {
+            listHomeRecyclerView.setHasFixedSize(true);
+            listHomeAdapter = new SubscriberAdapter(homeViewState.getUsers(), userClickListener);
```

Comment:
You're always creating a new adapter instance every time render method with users is invoked. Please initialize this as the rest of components and only set the users + invalidate adapter